pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -698,32 +698,51 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
 		return
 	}
+	currentPodMemoryConfig, err := pcm.GetPodCgroupConfig(pod, v1.ResourceMemory)
+	if err != nil {
+		klog.ErrorS(nil, "Unable to get pod cgroup memory config", "pod", pod.Name)
+		result.Fail(fmt.Errorf("unable to get pod cgroup memory config for pod %s", pod.Name))
+		return
+	}
+	currentPodCPUConfig, err := pcm.GetPodCgroupConfig(pod, v1.ResourceCPU)
```

Comment:
Since you're fetching these up-front, they no longer need to be fetched below (lines 785 & 820)